.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+      - branch-*
+      - dogfood-*
+  pull_request:
+  workflow_dispatch:
+    schedule:
+    - cron: "30 1 * * *" # Run daily at 1:30 AM UTC
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: github-ubuntu-latest-m  # Public repo
+    name: Build
+    permissions:
+      id-token: write  # Required for Vault OIDC authentication
+      contents: write  # Required for repository access and tagging
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - uses: jdx/mise-action@5ac50f778e26fac95da98d50503682459e86d566 # v3.2.0
+        with:
+          version: 2025.7.12
+      - name: Install Property Dump Plugin
+        run: |
+          mvn -f property-dump-plugin/pom.xml -B install
+      # Run setup-gradle to prevent validating the wrapper in
+      # integrationTests/src/test/resources/AndroidTestingBlueprintWithFeatureModule
```

Comment:
Is this a temporary fix? If so, can we add a FIXME comment and link to the ticket that we are waiting for to remove it?